bylaws/009-funding.md

```diff
@@ -6,48 +6,50 @@ PHP-FIG is, as it often happens in Open Source communities, an organization comp
+All three Secretaries, but only the Secretaries, MUST have full administrative access to PHP-FIG's OpenCollective account. They SHALL NOT use those permissions in a discretionary manner, except as stated explicitly in this bylaw.
 
-All monetary contributions will flow through Open Collective, so that any tax considerations will be handled automatically, and every financial decision will be published and publicly available through Open Collective's budgetary tools.
+All monetary contributions MUST flow through Open Collective, so that any tax considerations will be handled automatically, and every financial decision MUST be published and publicly available here and through Open Collective's budgetary tools.
 
 # Raising money
 
-PHP-FIG accepts donations through multiple channels.  Only the Secretaries may actively solicit donations from any party.  No other PHP-FIG member may do so.
+PHP-FIG SHALL NOT solicit donations from any party, especially from private individuals. The only allowed funding sources MUST be listed below.
 
 ### Through OpenCollective
 
-Direct contributions will be accepted through OpenCollective.
+Direct contributions MUST NOT be accepted through OpenCollective; the collective home page MUST be set up in a way that:
+ * all contribution sections and features are hidden and/or disabled, if possible;
+ * explicit wording will be visible reminding visitors of the PHP-FIG stance in the matter.
 
 ### Through TideLift
 
-Contributions will be accepted through [TideLift](https://tidelift.com/).  They will pay out to the OpenCollective account.
-
-### Through Thanks.dev
-
-Contributions will be accepted through [Thanks.dev](https://thanks.dev).  They will pay out to the OpenCollective account.
+Contributions will be accepted through [TideLift](https://tidelift.com/). All three Secretaries, but only the Secretaries, MUST have full administrative access to PHP-FIG's TideLift account, which MUST be configured to pay out to the Open Collective account. 
 
 ## Spending money
 
-Funds donated to the PHP-FIG are to be used for non-personnel operating expenses.  PHP-FIG does not pay individual contributors to PHP-FIG standards or working groups.
-All operating or capital expenses must be approved by an Approval Vote of the Core Committee.  The expense must include whether it is one time or recurring and at which frequency.  All expenses must be justified by their contribution to the mission of PHP-FIG.
+Funds donated to the PHP-FIG MUST NOT be used for non-personnel operating expenses.  PHP-FIG SHALL NOT pay individual contributors to PHP-FIG standards or working groups.
+All operating or capital expenses MUST be approved by an Approval Vote of the Core Committee.  The expense MUST include whether it is one time or recurring and at which frequency.  All expenses MUST be justified by their contribution to the mission of PHP-FIG.
```

Comment:
I the above line becomes `MUST be used for non-personnel operating expenses`, what would happen to capital expenses?